test/run.ts

```diff
@@ -72,8 +71,13 @@ async function runTest(list: any) {
 
   await Promise.all(
     set.map(async ({ obj, result }: any) => {
-      (obj.newoutput.ast = result), obj.file;
-      obj.newoutput.pretty = '✖ Soon to be open sourced'; //toJs(result);
+      obj.newoutput.ast = result;
+      obj.newoutput.pretty = printSourceFile(result, {
+        tabWidth: 2,
```

Comment:
'printSourceFile' should be a dummy function here because not implemented yet. Else things will break.

The printer options should be extracted and part of the "top options" in the .md file